controllers/workspace/condition.go

```diff
@@ -130,3 +132,12 @@ func getConditionIndexInOrder(condType dw.DevWorkspaceConditionType) int {
 	}
 	return -1
 }
+
+func capitalizeMsg(msg string) string {
```

Comment:
I think this function name could be unabbreviated as `capitalizeMessage(msg string)`.

Also, it's worth adding a simple function doc, something like:
`// Capitalizes the first character in the given string.`